### DISCUSSION TITLE
[Request] Make autocomplete easier

### DISCUSSION THREAD

**Discussion by peterstriz** [dineug/erd-editor#271] [Ideas]
The suggestions (autocomplete) are awesome and I can move with arrows, but I cannot use only keyboard to fill the values.

Idea: When adding datatype to column, use `Tab`/`Enter` to autocomplete

![image](https://user-images.githubusercontent.com/43370218/126337790-5b04f0b7-4239-4a97-94a2-034453340fa9.png)

**Comment by dineug** [comment]
It is currently supported by the right arrow key.
I'll consider applying for Enter.
